sdk/cosmos/azure-cosmos/docs/StatusCodes.md

```diff
@@ -0,0 +1,61 @@
+# Cosmos DB Java SDK - Status Codes
+
+## Overview
+
+Below is a list of the different status code / sub status code combinations that admins and/or developers can experience in their application or when looking at diagnostics/metrics when using the Cosmos DB Java SDK.
+Several of these error conditions would never surface to the application, because the SDK has built-in retry-logic to recover. But these status codes could show up in diagnostics and/or micrometer metrics. Common questions from users regarding the context of these status codes include::
+- what do these error conditions mean - even when they are automatically handled by the SDK?
+- how long should a certain error condition happen before self-recovery happens?
+
+The column "Expected to be transient" indicates whether this error condition is to be expected to always be seen for a short period of time only (and auto-mitigated within seconds or at least few minutes should happen in the SDK and/or the service). For error conditions that might not go away automatically the description contains additional context on what to watch out for if this error is seen for longer period of times. For errors that are supposed to be "transient" it will be necessary to file a support ticket, if the error condition exists for an extended period of time. The "Additional Info" column will also point to known issues if available and possible mitigations.
+
+## Out of scope
+This document is intentionally not going into details on how resilient applications should react  to certain error conditions (and whether/how retries are recommended). There is prescriptive guidance for developers around this located here: [Designing resilient applications with Azure Cosmos DB SDKs](https://learn.microsoft.com/azure/cosmos-db/nosql/conceptual-resilient-sdk-applications)
+
+## Status codes
+
+| Status code | Substatus code | Expected to be transient | Additional info |
+| -----------------:|-----------------------:|:-----------------------:|:----------------------|
+|200|0|No| `OK`|
+|201|0|No| `Created` - returned for createItem or upsertItem when a new document was created|
+|204|0|No| `No Content` - returned when no payload would ever be returned - like for delete operations|
+|207|0|No| `Multi-Status` - returned for bulk operations when some of the item operations have failed and others succeeded. The API allows checking status codes of item operations.|
+|304|0|No| `Not Modified` - will be returned for `ReadFeed` and `ChangeFeed` operations to indicate that there are no more changes|
```

Comment:
Technically this is only for Change Feed. ReadFeed appears in our metrics for Change Feed requests, but its not really a ReadFeed.